tests/integration/link/test_link_children.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import dash
+from dash.dependencies import Input, Output
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+@pytest.mark.DCC776
+def test_children001_default(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            dcc.Link(id="link1", href="/page-1"),
+            dcc.Location(id="url", refresh=False),
+            html.Div(id="content")
+        ]
+    )
+    @app.callback(Output("content", "children"), [Input("link1", "children")])
+    def display_children(children):
+        return children
+
+    dash_dcc.start_server(app)
+
+    href_as_children = dash_dcc.driver.execute_script(
+        '''
+        return document.getElementById("link1").text;
+        '''
+    )
```

Comment:
`dash_dcc.wait_for_text_to_equal("#link1", "/page-1")`

Reply:
Added in d25a0e4.

Reply:
That statement replaces the whole `href_as_children = ...` and `assert href_as_children == ...`

Reply:
I'm guessing that method acts as a wrapper of sorts to check the text? Interesting. Updated in da999d5.